TestFramework/Tests.Integration/Tests/DataStoreNetworkIntegrationTests.cs

```diff
@@ -1517,6 +1517,144 @@ where todo.Name.StartsWith("a to") || todo.Details.Equals("details for 2")
 		}
 
 		[Test]
+		public async Task TestNetworkStoreFindByQueryMultipleWhereClausesEquals()
```

Comment:
@tejasranade Completely agree.  I will look into how we can expose out the query string that is built, so that we can compare it against expected string without having to check against the results.